src/inject/utils/dom.ts

```diff
@@ -97,9 +97,10 @@ export function watchForNodePosition<T extends Node>(
             start = now;
             attempts = 1;
         }
-        if (prevSibling && prevSibling.parentNode !== parent) {
-            logWarn('Unable to restore node position: sibling was removed', node, prevSibling, parent);
-            stop();
+        if (prevSibling && (prevSibling.parentNode !== parent && prevSibling.parentNode !== null)) {
+            logWarn('Style was moved to another parent, moving node\'s position', node, prevSibling, parent);
```

Comment:
I'm not sure, but I think you cannot *move the position*, you can *change the position* or *restore the position*. Also, since `\'` looks a little bit weird, you can use a template string quotes and `'` without the escaping.